calico-enterprise/networking/configuring/custom-bgp-config.mdx

```diff
@@ -1,11 +1,17 @@
 ---
+description: Customize your BGP configuration.
 ---
 
-# Custom BGP Configuration
+# Custom BGP configuration
 
 ## Big picture
 
-Use customized BIRD configuration files to enable specialized use-cases.
+Customize BGP configurations for special use-cases.
+
+:::caution
+Customizing BGP configuration templates is not supported by Tigera for production because they are not guaranteed to work when upgrading. However, if you are considering custom templates for POCs and other temporary uses cases, please work with your Tigera Support representative.
```

Comment:
Is it okay to use "please"?

Reply:
Fixed. I was working with original text from PM so I didn't edit that out, but yes, it should come out. Good one!